docs/src/user/advanced.md

```diff
@@ -248,7 +248,7 @@ want to disable this checking.
 
 1. (preferred) Provide an extra `Val{false}()` argument to the differentiation function, e.g.
    ```julia
-   cfg = ForwarDiff.GradientConfig(g, x)
+   cfg = ForwardDiff.GradientConfig(g, x)
    ForwarDiff.gradient(f, x, cfg, Val{false}())
```

Comment:
Seems there's another typo in the line below:
```suggestion
   ForwardDiff.gradient(f, x, cfg, Val{false}())
```

Reply:
Thanks! It was a little early on my end, sorry for missing the second typo :)